### PR TITLE
fix(admin-welcome): issues when sign up for the Give newsletter #2322

### LINKED ISSUE #2322: fix(admin-welcome): issues when sign up for the Give newsletter
## Issue Overview
Please see the video for more information 
http://youtu.be/1srqmPwd5Cs?hd=1

## Expected Behavior
Email checking and change message after form is submitted successfully 

## Current Behavior
Email id is not checked 
form message show that you have to be  successfully subscibe to the mailing list 

## Steps to Reproduce (for bugs)
See the Video

## Todos
- [x] Tests
- [x] Documentation

## WordPress Environment
<details>
```
### WordPress Environment ###

Home URL: http://localgive.dev
Site URL: http://localgive.dev
WP Version: 4.8.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
Permalink Structure: /index.php/%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: 3
Table Prefix Status: Acceptable
Admin AJAX: Accessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: localgive.dev
TLS Connection: Connection uses TLS 1.2
TLS Connection: Probably Okay
Server Info: nginx/1.10.1
PHP Version: 7.0.3
PHP Post Max Size: 1,000 MB
PHP Time Limit: 900
PHP Max Input Vars: 3000
PHP Max Upload Size: 1,000 MB
cURL Version: ❌ 7.38.0, OpenSSL/1.0.1t - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.55 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 1.8.17
Upgraded From: 1.8.17
Test Mode: Enabled
Currency Code: USD
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: http://localgive.dev/index.php/donation-confirmation/
Failure Page: http://localgive.dev/index.php/donation-failed/
Donation History Page: http://localgive.dev/index.php/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, Offline Donation
Default Payment Gateway: Test Donation
PayPal IPN Verification: Enabled
PayPal IPN Notifications: N/A
Admin Email Notifications: Enabled
Donor Email Access: Enabled

### Session Configuration ###

Give Use Sessions: Enabled
Session: Enabled
Session Name: PHPSESSID
Cookie Path: /
Save Path: 
Use Cookies: Enabled
Use Only Cookies: Enabled

### Active Give Add-ons ###

Give - Fee Recovery: ❌ Unlicensed – by WordImpress – 1.3

### Other Active Plugins ###

Query Monitor: by John Blackbourn – 2.17.0

### Inactive Plugins ###

Debug Bar Console: by koopersmith – 0.3
Debug Bar Cron: by Zack Tollman, Helen Hou-Sandi – 0.1.2
Debug Bar Extender: by Thorsten Ott, Automattic – 0.5
Developer: by Automattic – 1.2.6
Duplicate Post: by Enrico Battocchi – 3.2
Give - 2Checkout Gateway: by WordImpress – 1.0.2
Give - AmeriCloud Payments: by AmeriCloud Solutions, Inc. – 1.2.0
Give - Authorize.net Gateway: by WordImpress – 1.3
Give - CCAvenue Gateway: by WordImpress – 1.0
Give - Currency Switcher: by WordImpress – 1.0.0
Give - Email Reports: by WordImpress – 1.0.1
Give - Form Field Manager: by WordImpress – 1.2.4
Give - Gift Aid: by WordImpress – 1.0.0
Give - Manual Donations: by WordImpress – 1.2
Give - PayPal Pro Gateway: by WordImpress – 1.1.4
Give - PDF Receipts: by WordImpress – 2.0.7
Give - PDF Receipts: by WordImpress – 2.0.7
Give - Per Form Emails: by WordImpress – 1.0.2
Give - Razorpay: by WordImpress – 1.1.0
Give - Recurring Donations: by WordImpress – 1.4
Give - Stripe Gateway: by WordImpress – 1.5
Give - Tributes: by WordImpress – 1.2
Give - WePay Gateway: by WordImpress – 1.3.1
Give - WP All Import Addon: by Ben Meredith – 1.0
Give - Zapier: by WordImpress – 1.1.2
Give Cutom Plugin: by  –
Give to ZBS CRM Connector: by https://zerobscrm.com – 1.0.1
Hookr: by @explodybits – 1.0.0 (beta)
Loco Translate: by Tim Whitlock – 2.0.16
Log Deprecated Notices: by Andrew Nacin – 0.4
Log Viewer: by Markus Fischbacher – 14.05.04
Redis Object Cache: by Till Krüss – 1.3.5
Rewrite Rules Inspector: by Daniel Bachhuber, Automattic – 1.2.1
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com, DavidAnderson – 1.13.12
User Switching: by John Blackbourn – 1.3.0
WooCommerce: by Automattic – 3.2.3
WordPress Importer: by wordpressdotorg – 0.6.3
WP Downgrade | Specific Core Version: by Reisetiger – 1.1.4
WP Rollback: by WordImpress – 1.5
WP Super Cache - Clear all cache: by Apasionados.es – 1.4

### Theme ###

Name: Twenty Sixteen
Version: 1.3
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme

```
</details>

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR to fix #2322 

## How Has This Been Tested?
Test by sending mail from admin welcome screen

## Screenshots (jpeg or gifs if applicable):
image

image

image

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/misc-functions.php [hunk 1]
```diff
@@ -580,9 +579,9 @@ function give_get_newsletter() {
 
 		<form action="//givewp.us3.list-manage.com/subscribe/post?u=3ccb75d68bda4381e2f45794c&amp;id=12a081aa13"
 		      method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate"
-		      target="_blank" novalidate>
+		      target="_blank">
 			<div class="give-newsletter-confirmation">
-				<p><?php esc_html_e( 'Thanks for Subscribing!', 'give' ); ?> :)</p>
+				<p><?php esc_html_e( 'A confirmation mail is being sent to your email ID. Kindly conform your subscription over there! :)', 'give' ); ?></p>
```
Comment (raftaar1191) [inline includes/misc-functions.php]: @DevinWalker Need you help to update this text

**Comment by raftaar1191** [comment]
@DevinWalker @ravinderk 

Need your help to update this text. I am moving this issues Blocker Pipline 

Text Link: https://github.com/WordImpress/Give/pull/3325#discussion-diff-193223425R584

File: includes/misc-functions.php (hunk 1 above)
Comment (kevinwhoffman) [reply includes/misc-functions.php]: @raftaar1191 Please use the following text:
> To complete your subscription, click the confirmation link in your email. Thank you!

**Comment by kevinwhoffman** [comment]
@raftaar1191 I provided the text for the confirmation message. Moving this back into your sprint backlog now.

**Comment by raftaar1191** [comment]
@kevinwhoffman PR update as suggested by you I am moving this Issues in the Ready for QA pipeline for review and merging

**Pull request merged**